### PR TITLE
Add biolink-model version to x-translator

### PR DESCRIPTION
**Pull request by vdancik**

### CODE REVIEW

**Review by edeutsch — APPROVED** [review]

**Comment by patrickkwang** [comment]
Looks like branching got wonky here. This may just need to be rebased onto v1.1.x.

**Review by ehinderer — APPROVED** [review]

**Review by bill-baumgartner — APPROVED** [review]

File: TranslatorReasonerAPI.yaml [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   termsOfService: INSERT-URL-HERE
+  x-trapi:
+    version: 1.1.0
+  x-translator:
+    component: INSERT-EITHER-ARA-OR-KP-HERE
+    team: INSERT-TRANSLATOR-TEAM-NAME-HERE
+    biolink_model_version: INSERT-BIOLINK-MODEL-VERSION-HERE
```
Comment (colleenXu) [inline TranslatorReasonerAPI.yaml:17]: Change this to match translator_extension schema?

The translator_extension schema currently uses [biolink-version](https://github.com/NCATSTranslator/translator_extensions/blob/62755f92c82d6aafa2ae55c394434640ebb9a0dd/x-translator/smartapi_x-translator_schema.json#L52) as the property name, with acceptable values following the official biolink releases: so versions like '1.6.1', '1.6.0'...

File: TranslatorReasonerAPI.yaml [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
   contact:
     email: edeutsch@systemsbiology.org
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   termsOfService: INSERT-URL-HERE
+  x-trapi:
+    version: 1.1.0
+  x-translator:
+    component: INSERT-EITHER-ARA-OR-KP-HERE
```
Comment (colleenXu) [inline TranslatorReasonerAPI.yaml:15]: Team is not correctly structured here, and the component description doesn't include all the actual allowed values. See #217 for my recommendations of how to specify x-trapi and x-translator.

**Review by colleenXu — CHANGES_REQUESTED** [review]
See my comments for the changes I requested/issues I have.

**Pull request merged**